frb_codegen/src/generator/dart/mod.rs

```diff
@@ -327,7 +328,9 @@ fn generate_dart_implementation_body(spec: &DartApiSpec, config: &Opts) -> Acc<D
         ),
         io: format!(
             "class {plat} extends FlutterRustBridgeBase<{wire}> {{
-                {plat}(ffi.DynamicLibrary dylib) : super({wire}(dylib));",
+                {plat}(ffi.DynamicLibrary dylib) : super({wire}(dylib)) {{
+                    DartApiDl.initApi(dylib);
```

Comment:
As mentioned earlier, would be great to have lazy initialization, such that people not using Dart Opaque will not have any cost at all.

(not reviewed whole code yet, so if it already have lazy init, ignore this comment)